cluster/addons/dns/kube2sky/Makefile

```diff
@@ -4,7 +4,12 @@
 
 .PHONY: all kube2sky container push clean test
 
-TAG = 1.8
+# By default, pushes to a githash.  To override:
+# make push TAG=<version>
+# Next TAG=1.9
+
+TAG = $(shell git log -1 --format="format:%H")
```

Comment:
My version is unfortunately kubectl specific-- maybe there's an easy way to compute the version string without actually calling one of our binaries-- that would be great, but I didn't find one digging through our shell code...